crypto/src/main/java/net/corda/v5/crypto/CompositeKeyNodeAndWeight.java

```diff
@@ -13,10 +12,11 @@ public final class CompositeKeyNodeAndWeight {
     private final int weight;
 
     /**
-     * Construct a key.
+     * Creates a new {@code CompositeKeyNodeAndWeight} for
+     * the specified key and weight for the key.
      *
-     * @param node   A public key.
-     * @param weight The weight for that key, must be greater than zero.
+     * @param node   A public key
+     * @param weight The weight for that key, must be greater than zero
```

Comment:
We should describe what the weight does

Reply:
Something like

@param weight The weight for that key, must be greater than zero. The weight controls the contribution a key makes to reaching the threshold that specifies whether the composite key is regarded as fulfilled.

Reply:
Agreed, perhaps better address this under a different JIRA?

Reply:
Ok

Reply:
Raised [JIRA](https://r3-cev.atlassian.net/browse/CORE-13722) for it.